test/functional/apps/discover/_discover.js

```diff
@@ -106,9 +106,9 @@ export default function ({ getService, getPageObjects }) {
         await PageObjects.discover.brushHistogram();
 
         const newDurationHours = await PageObjects.timePicker.getTimeDurationInHours();
-        expect(Math.round(newDurationHours)).to.be(108);
+        expect(Math.round(newDurationHours)).to.be(25);
         const rowData = await PageObjects.discover.getDocTableField(1);
-        expect(rowData).to.have.string('Sep 22, 2015 @ 23:50:13.253');
+        expect(Date.parse(rowData)).to.be.within(Date.parse('Sep 20, 2015 @ 22:00:00.000'), Date.parse('Sep 20, 2015 @ 23:30:00.000'));
```

Comment:
Should we document in the code why we need this range?

Reply:
We didn't do it in the past. I will ask @LeeDr  to run it on Windows to get exact values and we can tweak it later with more precise range

Reply:
and comment explanation with specific values:)